utmap-client/src/components/pages/CreateEventPage.js

```diff
@@ -66,13 +66,18 @@ function CreateEventPage({onClose, addEvent}) {
 		//Send info???
 		startDate.setSeconds(0,0);
 		endDate.setSeconds(0,0);
-		const eventForm = {
-			title, 
-			startDate: startDate.toLocaleString('en-US', { timeZone: 'America/New_York' }), 
-			endDate: endDate.toLocaleString('en-US', { timeZone: 'America/New_York' }),
-			location, sublocation, description};
-		addEvent(eventForm);
-		onClose();
+		
+		if (startDate > endDate){
```

Comment:
I don't think a simple comparison will work. 
Eg.
```JavaScript
startDate = "10/5/2021, 6:00:00 PM";
endDate = "1/5/2021, 6:00:00 PM";
startDate > endDate  == True
```

Reply:
Do `Date.parse(startDate) > Date.parse(endDate)` to convert them into a number instead  
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

Reply:
It does seem to work? Unless it's bad form to not do a number conversion first

![image](https://user-images.githubusercontent.com/25747480/107988506-2f0f9b80-6f9e-11eb-8037-20e2ef21fc7c.png)

Reply:
Ahhh my bad I thought you converted to string and then compared.